CHANGELOG.md

```diff
@@ -2,29 +2,25 @@
 
```

Comment:
How does husky mess up the changelog? By having husky we can check for ts and linting errors before we push to avoid clogging the commit history with ts and lint fixes.